### PR TITLE
Completed es6-template-literals exercise. - Charlie McCanna

### PR DESCRIPTION
**Pull request by cgmccanna**
es6-template-literals

### CODE REVIEW

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]
@cgmccanna You missed one of the concatenations:

---

image

**Comment by cgmccanna** [comment]
es6-template-literals-updated

File: es6-template-literals/index.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    const age = 25;
+
+    const bio = `My name is ${firstName} ${lastName} and I am ${age} years old.`;
+
+    console.log('Bio:', bio);
+
+    const fruit = 'apple';
+    const item = 'pen';
+
+    const applePen = fruit + item;
```
Comment (TimelikeClosure) [inline es6-template-literals/index.html]: This concatenation should be converted to a template literal.

File: es6-template-literals/index.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+    console.log(`I have a ${item}`);
+    console.log(`I have an ${fruit}`);
```
Comment (TimelikeClosure) [inline es6-template-literals/index.html]: The original `console.log`s were passed multiple arguments, which involved no concatenation, so they should not have been modified:

---

image

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]

**Comment by cgmccanna** [comment]
es6-template-literals-updated

**Review by TimelikeClosure — APPROVED** [review]

**Pull request merged**